fetch.bs

```diff
@@ -2728,26 +2730,63 @@ functionality.
 <p>Each <a>environment settings object</a> has an associated
 <dfn export id=concept-fetch-group for=fetch>fetch group</dfn>.
 
-<p>A <a for=fetch>fetch group</a> holds an ordered list of
-<dfn lt="fetch record" export for="fetch group" id=concept-fetch-record>fetch records</dfn>.
+<p>A <a for=fetch>fetch group</a> has an associated
+<dfn lt="fetch record" export for="fetch group" id=concept-fetch-record>fetch records</dfn>,
+a <a for=/>list</a> of <a for="/" data-lt="fetch record">fetch records</a>.
 
-<p>A <a for="fetch group">fetch record</a> has an associated
-<dfn export for="fetch record" id=concept-fetch-record-request>request</dfn> (a
-<a for=/>request</a>).
+<p>A <a for=fetch>fetch group</a> has an associated
+<dfn export for="fetch group" id=concept-defer=fetch-record>deferred fetch records</dfn>,
+a <a for=/>list</a> of <a data-lt="deferred fetch record">deferred fetch records</a>.
 
-<p>A <a for="fetch group">fetch record</a> has an associated
-<dfn export for="fetch record" id=concept-fetch-record-fetch>controller</dfn> (a
-<a for=/>fetch controller</a> or null).
+<p>A <dfn export>fetch record</dfn> is a [=struct=]. It has the following items:
+
+<dl>
+ <dt><dfn export for="fetch record">request</dfn>
+ <dd>A <a for=/>request</a>.
+
+ <dt><dfn export for="fetch record">controller</dfn>
+ <dd>A <a for=/>fetch controller</a> or null.
+</dl>
+<hr>
+
+<p>A <dfn>deferred fetch record</dfn> is a <a for=/>struct</a> used to maintain state needed
+to invoke a fetch at a later time, e.g., when a {{Document}} object is unloaded or becomes
+not <a for=Document>fully active</a>. It has the following <a for=struct>items</a>:
+
+<dl>
+ <dt><dfn for="deferred fetch record">request</dfn>
+ <dd>A <a for=/>request</a>.
+
+ <dt><dfn for="deferred fetch record">notify invoked</dfn>
+ <dd>An algorithm accepting no arguments.
+
+ <dt><dfn for="deferred fetch record">invoke state</dfn> (default "<code>pending</code>")
+ <dd>"<code>pending</code>", "<code>fetching</code>", "<code>done</code>", or
```

Comment:
> `done`

There were [extensive discussion](https://github.com/WICG/pending-beacon/issues/75) before about `sent` state (or similarly `done` here), and the conclusion is that we should not use this term in this API, as a fetchLater request may be processed in unload or in bfcached and once it is done the response may not be able to get back to the JS context.

Although this is just an internal state, it still requires knowledge of when a request is done from the view of a renderer (document), which is difficult to get right in async.